Modules/SearchCityViewController/Models/FormattedCityWeatherModel.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CityWeatherInternal.swift
+//  WeatherDemoApp
+//
+//  Created by ahmed on 04/04/2024.
+//
+
+import Foundation
+import RealmSwift
+
+final class FormattedCityWeatherModel: Object {
```

Comment:
Formatted? Persisted models should be plain and you move this formatting thing to the view model instead.